lib/srv/desktop/rdp/rdpclient/client_common.go

```diff
@@ -32,16 +32,18 @@ type Config struct {
 	// UserCertGenerator generates user certificates for RDP authentication.
 	GenerateUserCert GenerateUserCertFn
 
+	// Login is the Windows login name.
+	Login string
+	// Width is the initial screen width.
+	Width string
```

Comment:
I might call these `InitialWidth` and `InitialHeight` given that we will soon support resizing.

Also, should they be strings?